app/views/handlers/individual_response.py

```diff
@@ -132,6 +141,58 @@ def form(self):
     def get_context(self):
         return build_question_context(self.rendered_block, self.form)
 
+    def _get_fulfilment_code(self, mobile_number=None):
+        fulfilment_codes_map_for_sms = {
+            GB_ENG_REGION_CODE: "UACITA1",
+            GB_WLS_REGION_CODE: "UACITA2B",
+            GB_NIR_REGION_CODE: "UACITA4",
+        }
+        fulfilment_codes_map_for_postal = {
+            GB_ENG_REGION_CODE: "P_UAC_UACIP1",
+            GB_WLS_REGION_CODE: "P_UAC_UACIP2B",
+            GB_NIR_REGION_CODE: "P_UAC_UACIP4",
+        }
+
+        region_code = self._metadata["region_code"]
+        return (
+            fulfilment_codes_map_for_sms[region_code]
+            if mobile_number
+            else fulfilment_codes_map_for_postal[region_code]
+        )
```

Comment:
If it was more explicit I would be tempted to put it in one dict, something like 

```
fulfilment_codes = {
            "sms": {
                GB_ENG_REGION_CODE: "UACITA1",
                GB_WLS_REGION_CODE: "UACITA2B",
                GB_NIR_REGION_CODE: "UACITA4",
            },
            "postal": {
                GB_ENG_REGION_CODE: "P_UAC_UACIP1",
                GB_WLS_REGION_CODE: "P_UAC_UACIP2B",
                GB_NIR_REGION_CODE: "P_UAC_UACIP4",
            }

        }
        region_code = self._metadata["region_code"]
        return fulfilment_codes.get(type, {}).get(region_code)
```

Reply:
My mistake, this should have not passed the number. Have changed to be as above.